app/models/user.js

```diff
@@ -2,7 +2,8 @@
 import Model, { attr, hasMany } from '@ember-data/model';
 
 export default class UserModel extends Model {
-  @attr('string') name;
+  @attr('string') firstName;
+  @attr('string') lastName;
   @attr('string') description;
```

Comment:
What about adding getter `name` here ? It would concat first and last name then in the template still we can use `this.user.name`

Reply:
> Like this?

Yes exactly :ok_hand: 
Imo `name` is perfect candidate for getter in model here.

Reply:
Hi Piotr - Should I change my code back to use this approach or leave as is in this case?